servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -2459,6 +2459,24 @@ void RendererSceneRenderRD::render_buffers_configure(RID p_render_buffers, RID p
 		p_internal_width = p_width;
 	}
 
+	if (p_use_taa) {
+		// Use negative mipmap LOD bias when TAA is enabled to compensate for loss of sharpness.
+		// This restores sharpness in still images to be roughly at the same level as without TAA,
+		// but moving scenes will still be blurrier.
+		p_texture_mipmap_bias -= 0.5;
+	}
+
+	if (p_screen_space_aa == RS::VIEWPORT_SCREEN_SPACE_AA_FXAA) {
+		// Use negative mipmap LOD bias when FXAA is enabled to compensate for loss of sharpness.
+		// This restores sharpness to an extent, but is more conservative
+		// compared to the mipmap bias used for TAA as FXAA does not provide
+		// temporal antialiasing. A strong negative mipmap bias can cancel out
+		// the blurriness almost entirely, but at the cost of grainy textures at
+		// a distance.
+		// If both TAA and FXAA are enabled, combine their negative LOD biases together.
```

Comment:
```suggestion
		// If both TAA and FXAA are enabled, combine their negative LOD biases together.
```
I would remove this as it doesn't add anything for the future reader. It is vague and comparing it to the TAA bias which is immediately above is unhelpful.